hostscripts/clouddata/syncSLErepos

```diff
@@ -122,8 +122,8 @@ for version in 9 8 7 6; do
         # Ardana flavor
         if [ "$version" != "8" ] || [ "$arch" = "x86_64" ]; then
             if [ -z "$sync_from_ibs" ]; then
-                rsync_and_unpack_iso ${ibs_source}/Devel\:/Cloud\:/$version/images/iso/SUSE-OPENSTACK-CLOUD-$version-$arch-Media1.iso /srv/nfs/repos/$arch/SUSE-OpenStack-Cloud-$version-devel
-                rsync_and_unpack_iso ${ibs_source}/Devel\:/Cloud\:/$version\:/Staging/images/iso/SUSE-OPENSTACK-CLOUD-$version-$arch-Media1.iso /srv/nfs/repos/$arch/SUSE-OpenStack-Cloud-$version-devel-staging
+                rsync_and_unpack_iso ${ibs_source_nue}/Devel\:/Cloud\:/$version/images/iso/SUSE-OPENSTACK-CLOUD-$version-$arch-Media1.iso /srv/nfs/repos/$arch/SUSE-OpenStack-Cloud-$version-devel
```

Comment:
with this change, you don't need the `if` anymore, because it looks like the if and else blocks are the same now

Reply:
ok, I personally would have preferred the if condition to remain in tact, but we can remove it..